src/legacy/dom-module.html

```diff
@@ -57,6 +75,7 @@
         modules[id] = this;
         lcModules[id.toLowerCase()] = this;
         this._styleOutsideTemplateCheck();
+        this._setAssetpath();
```

Comment:
Yes, it could just be a getter. `Polymer.Element.resolveUrl` would need to be updated accordingly.